examples/41_path-alias/waku.config.ts

```diff
@@ -0,0 +1,23 @@
+import { fileURLToPath } from 'node:url';
+import { defineConfig } from 'waku/config';
+import tsconfigPaths from 'vite-tsconfig-paths';
+
+const getConfig = () => ({
+  ssr: {
+    external: ['vite-tsconfig-paths'],
+  },
+  plugins: [
+    tsconfigPaths({ root: fileURLToPath(new URL('.', import.meta.url)) }),
+  ],
+});
+
+export default defineConfig({
+  unstable_viteConfigs: {
+    'dev-main': getConfig,
+    'dev-rsc': getConfig,
+    'build-analyze': getConfig,
+    'build-server': getConfig,
+    'build-ssr': getConfig,
+    'build-client': getConfig,
+  },
+});
```

Comment:
would it make sense to have a "common" field that applies to the viteConfig in all environments/steps? 

This seems like a good example for when that would be useful

Reply:
Yeah, it's possible.